packages/components/src/Menu/Menu.css

```diff
@@ -8,27 +8,53 @@
 }
 
 .menu {
-  position: absolute;
+  position: fixed;
   right: 0;
+  bottom: 0;
+  left: 0;
   z-index: var(--elevation-menu);
-  width: calc(var(--base-unit) * 12.5);
-  max-height: 300px;
-  box-shadow: var(--shadow-base);
+  width: 100%;
+  max-height: 72vh;
   padding: var(--space-small);
+  padding-bottom: env(safe-area-inset-bottom);
```

Comment:
what is `env()`? I think this might be a `scss` function that got sucked over from sg1

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/env

Reply:
interesting. never heard of this

Reply:
[env()](https://developer.mozilla.org/en-US/docs/Web/CSS/env) is a real thing! 😆  This is for the notch / no button iPhones/other devices that move towards that pattern of screen-based home interactions.